pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.pkgproj

```diff
@@ -40,10 +40,6 @@
         <Exclude>Compile</Exclude>
       </Dependency>
     </ItemGroup>
-
-    <PropertyGroup>
-    <VersionSuffix>-rc3-$(CLIBuildVersion)</VersionSuffix>
```

Comment:
Yes it means it will start getting a `-00` as well. The host packages were already getting that as well. Today we are using commits but it isn't clear to me yet if that will remain the way we do it. If we switch this to full BuildTools for example that might change. Even with commit versioning there may be cases were you need to spin multiple builds from the same commit.